R/parameters.R

```diff
@@ -249,6 +249,8 @@
 #' model prophylaxis for each drug; default = turned off
 #' * drug_prophylaxis_scale - a vector of scale parameters for weibull curves to
 #' model prophylaxis for each drug; default = turned off
+#' * drug_hypnozoite_efficacy - a vector of efficacies targetting hypnozoites; default = turned off
+#' * drug_hypnozoite_prophylaxis - a vector of prophylaxis against hypnozoite batch formation (days)
```

Comment:
I wonder if we want this description to pertain to processes as modelled (as here), or to the biology they are trying to capture. An example of the latter would be something like:
```
drug_hypnozoite_prophylaxis - a vector of prophylaxis against liver-stage hypnozoite infection (days)
```
What do you think?

Reply:
I just came across this before I saw your comment. These should be shape and scale parameters, rather than a single prophylaxis one. I've corrected this in the next push.